README.md

```diff
@@ -1,25 +1,54 @@
-# Pod.Cast 🎱 🐋
+# OrcaHello: A real-time AI-assisted killer whale notification system 🎱 🐋
```

Comment:
To help reviewers, the bulk of additions are in this file @akashmjn @micya @scottveirs